main.go

```diff
@@ -58,6 +59,39 @@ func (l *ParsedList) toPlainText(listName string) error {
 	return nil
 }
 
+func (l *ParsedList) toGfwList() error {
+	var entryBytes []byte
+	entryBytes = append(entryBytes, []byte("[AutoProxy 0.2.9]\n")...)
+	for _, entry := range l.Entry {
+		switch entry.Type {
+		case "domain":
+			entryBytes = append(entryBytes, []byte("||"+entry.Value+"\n")...)
+		case "full":
+			entryBytes = append(entryBytes, []byte("|http://"+entry.Value+"\n")...)
+			entryBytes = append(entryBytes, []byte("|https://"+entry.Value+"\n")...)
+		case "keyword":
+			entryBytes = append(entryBytes, []byte(entry.Type+entry.Value+"\n")...)
```

Comment:
Sorry, I am not an expert at ABP rules, could you please explain why you need a `entry.Type`(which should always === "keyword") at the beginning of rule?